packages/http/src/http.ts

```diff
@@ -1,31 +1,31 @@
 import axios from 'axios';
 
 const authToken = {
-  access: sessionStorage.getItem('accessToken'),
-  refresh: sessionStorage.getItem('refreshToken'),
+  access: typeof window !== 'undefined' ? sessionStorage.getItem('accessToken') : null,
+  refresh: typeof window !== 'undefined' ? sessionStorage.getItem('refreshToken') : null,
```

Comment:
import { isServer } from '@toss/utils';


```suggestion
  access: isServer() && sessionStorage.getItem('accessToken'),
  refresh: isServer() && sessionStorage.getItem('refreshToken'),
```

은 어떠신가요